### ISSUE TITLE
Unexpected warning “Missing docstring” on potemkin's `import-var`

### ISSUE THREAD

**Issue by serioga** [borkdude/clj-kondo#935]
**version**

clj-kondo v2020.06.21

**platform**

Windows 10

**problem**

Unexpected warning “Missing docstring” on potemkin's `import-var`.

**repro**

```clojure
(ns dev.clj-kondo
  (:require [potemkin :refer [import-vars]]))

(import-vars [clojure.string blank?, starts-with?, ends-with?, includes?])
```

```
clj-kondo --lint src;dev
dev\dev\clj_kondo.clj:4:1: warning: Missing docstring.
```
